krun/results.py

```diff
@@ -92,6 +99,15 @@ def write_to_file(self):
             f.write(json.dumps(to_write,
                                indent=1, sort_keys=True, encoding='utf-8'))
 
+    def add_instrumentation_data(self, bench_key, inst_dct):
+        """Record instrumentation data into results object."""
+
+        for inst_key, v in inst_dct.iteritems():
+            # keys in dict are arbitrary, so we may need to scaffold a new list
```

Comment:
I don't understand this comment (is it suggesting the keys are in an arbitrary order? or ...?).